src/wanna/core/services/docker.py

```diff
@@ -338,29 +326,36 @@ def remove_image(image: Image, force=False, prune=True) -> None:
         """
         docker.image.remove(image, force=force, prune=prune)
 
-    @staticmethod
     def construct_image_tag(
-        registry: str,
-        project: str,
-        repository: str,
+        self,
         image_name: str,
-        registry_suffix: str,
-        versions: List[str] = ["latest"],
     ):
         """
         Construct full image tag.
         Args:
-            registry:
-            project:
-            repository:
             image_name:
-            versions:
 
         Returns:
             List of full image tag
         """
-
-        return [f"{registry}/{registry_suffix}{project}/{repository}/{image_name}:{version}" for version in versions]
+        versions = [self.version, "latest"]
+        return [
+            f"{self.docker_registry}/{self.docker_registry_suffix}{self.docker_project_id}/"
+            f"{self.docker_repository}/{self.wanna_project_name}/{image_name}:{version}"
+            for version in versions
+        ]
+
+    def build_container_and_get_image_url(self, docker_image_ref: str, skip_build: bool = False) -> str:
+        if skip_build:
+            docker_image_model = self.find_image_model_by_name(docker_image_ref)
+            tags = self.construct_image_tag(image_name=docker_image_model.name)
+            image_url = tags[0]
+        else:
+            image_tag = self.get_image(docker_image_ref=docker_image_ref)
+            if image_tag[1]:
```

Comment:
this will raise an exception `IndexError: list index out of range` if image_tag does not have such index. 

Is should not happen from looking at `self.get_image` but worth a tweak. WDYT